src/figuretype/animal.c

```diff
@@ -142,10 +142,7 @@ void figure_sheep_action(figure *f)
             if (f->wait_ticks > 400) {
                 f->wait_ticks = f->id & 0x1f;
                 f->action_state = FIGURE_ACTION_197_HERD_ANIMAL_MOVING;
-                f->destination_x = m->destination_x
-                    + formation_layout_position_x(FORMATION_HERD, f->index_in_formation);
-                f->destination_y = m->destination_y
-                    + formation_layout_position_y(FORMATION_HERD, f->index_in_formation);
+                formation_layout_position(FORMATION_HERD, f->index_in_formation, m, &f->destination_x, &f->destination_y);
```

Comment:
Line length is a bit too long (> 120 chars), suggestion to break it up like this. Also do the same for the other usages in the zebra and wolf functions.
```suggestion
                formation_layout_position(FORMATION_HERD, f->index_in_formation, m,
                    &f->destination_x, &f->destination_y);
```

Reply:
Since we're only using this for animals now, the FORMATION_HERD argument is unnecessary and the char count goes back down below 120.